src/cache.rs

```diff
@@ -252,6 +282,18 @@ impl Cache {
                             })?;
                         self.transactions.insert(key, transaction);
                     }
+                    CACHE_TYPE_KEY_VALUE => match key.as_str() {
```

Comment:
Could you explain the use of this new `KEY_VALUE` for cache?

Reply:
Yeah, I guess I wanted to avoid having to have a different cache entry for each individual static value entity, since there's ever only a single value for many entities like `bridge_addresses`. Will add a comment.